internal/termdbms/list/keys.go

```diff
@@ -0,0 +1,101 @@
+package list
+
+import "github.com/charmbracelet/bubbles/key"
+
+// KeyMap defines keybindings. It satisfies to the help.KeyMap interface, which
+// is used to render the menu menu.
+type KeyMap struct {
+	// Keybindings used when browsing the list.
+	CursorUp        key.Binding
+	CursorDown      key.Binding
+	NextPage        key.Binding
+	PrevPage        key.Binding
+	GoToStart       key.Binding
+	GoToEnd         key.Binding
+	Filter          key.Binding
+	ClearFilter     key.Binding
+	DeleteSelection key.Binding
+
+	// Keybindings used when setting a filter.
+	CancelWhileFiltering key.Binding
+	AcceptWhileFiltering key.Binding
+
+	// Help toggle keybindings.
+	ShowFullHelp  key.Binding
+	CloseFullHelp key.Binding
+
+	// The quit keybinding. This won't be caught when filtering.
+	Quit key.Binding
+
+	// The quit-no-matter-what keybinding. This will be caught when filtering.
+	ForceQuit key.Binding
+}
+
+// DefaultKeyMap returns a default set of keybindings.
+func DefaultKeyMap() KeyMap {
+	return KeyMap{
+		DeleteSelection: key.NewBinding(
+			key.WithKeys("r"),
+			key.WithHelp("r", "remove selection")),
+		// Browsing.
+		CursorUp: key.NewBinding(
+			key.WithKeys("up", "k", "w"),
```

Comment:
Removed some of the shortcuts to avoid conflict https://github.com/hazelcast/hazelcast-commandline-client/pull/65/commits/d9298f83c3b09a1db0760133e3d6224e87a5525e